core/lib/multivm/src/versions/vm_fast/vm.rs

```diff
@@ -537,6 +538,7 @@ where
         batch_env: L1BatchEnv,
         system_env: SystemEnv,
         storage: StoragePtr<StorageView<S>>,
+        _pubdata_builder: Option<Rc<dyn PubdataBuilder>>,
```

Comment:
Does it mean that the new VM doesn't correctly build pubdata now? How difficult would be to support it?

Reply:
Yes, not supported for now. It is easy to support, basically duplicate changes from old_vm